package.json

```diff
@@ -78,7 +78,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.0",
-    "cozy-ui": "^3.0.0-beta17",
+    "cozy-ui": "^3.0.0-beta18",
```

Comment:
Did you check that nothing broke? Especially form elements if there is some, because some classes changed with beta18.